mnefun/_mnefun.py

```diff
@@ -949,7 +949,6 @@ def run_sss_locally(p, subjects, run_indices):
             raw.fix_mag_coil_types()
             raw = filter_chpi(raw)
 
-            assert isinstance(p.trans_to, string_types)
             if p.trans_to is 'median':
```

Comment:
wdym how it got in? most likely I got it in, you have a check for this or what? 

I rm it bc (i) `trans_to` may also be an `np.array` (ii) thought the `if` statement should be enough, no? I can go with

```
assert isinstance(p.trans_to, (string_types, np.ndarray)) 
             if p.trans_to == 'median':
```
 ?